### ISSUE TITLE
Remove dates from item titles

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#189]
<img width="1033" alt="Screen Shot 2022-11-10 at 13 13 48" src="https://user-images.githubusercontent.com/1328900/201178005-2006859e-32ad-439c-a4ec-909c291cce76.png">

1. The app is configured to display titles together with dates. Not sure if we are indexing multiple fields into title, or displaying it in app code. The EAD does not combine the fields. We never ever want to display these datetimes in the title for any object.
2. Dates should be displayed in the component view for that item; We also need to format them to not be ugly strings. Many of the dates with have T00:00:00 -- all zeroes. We do not want to display that. However, some dates have real time values. We do want to display those.

```xml

            <c id="aspace_ef8c360c96645b3f0dbaa998d678c6da" level="item">
              <did>
                <unittitle>Defense for Hess. Part 1</unittitle>
                <unitid>H-5260_1732A_1</unitid>
                <physdesc altrender="whole">
                  <extent altrender="materialtype spaceoccupied">1 item(s)</extent>
                  <physfacet>1899-12-31T00:16:28+00:00</physfacet>
                </physdesc>
                <unitdate normal="1946-04-01T00:00:00+00:00/1946-04-01T00:00:00+00:00" type="inclusive">1946-04-01T00:00:00+00:00</unitdate>
                <unitdate normal="1946-04-01T10:10+01:00/1946-04-01T10:25+01:00" type="inclusive">1946-04-01T10:10+01:00-1946-04-01T10:25+01:00</unitdate>
                
                <dao xlink:actuate="onRequest" xlink:audience="external" xlink:href="https://purl.stanford.edu/mn513gx2686" xlink:show="new" xlink:title="Defense for Hess. Part 1, 1946-04-01T00:00:00+00:00" xlink:type="simple">
                  <daodesc>
                    <p>Defense for Hess. Part 1, 1946-04-01T00:00:00+00:00</p>
                  </daodesc>
                </dao>
              </did>

            </c>
```

**Comment by marlo-longley** [comment]
@laurensorensen

**Comment by marlo-longley** [comment]
Filter out T0's at the stage before it ever gets into EAD.

**Comment by thatbudakguy** [comment]
item 2 above will be resolved by #301

**Comment by marlo-longley** [comment]
Item 1 above blocked by https://github.com/projectblacklight/arclight/pull/1349 -- once we can customize the title in VT, we can remove the date.
